kolibri/core/assets/src/content-renderer.vue

```diff
@@ -70,7 +72,7 @@
     created() {
       this.findRendererComponent();
       // This means this component has to be torn down on channel switches.
-      this.$watch('id', this.findRendererComponent);
+      this.$watch('files', this.findRendererComponent);
```

Comment:
Because `id` was bound to the route parameter, this was not updating once content had fetched. Hoping that `files`is more predictably changing when the content should rerender.